[wdspec] add `bluetooth` module

sadym-chromium opened the pull request:
Add BiDi [bluetooth module](https://webbluetoothcg.github.io/web-bluetooth) and some wdspec tests.

sadym-chromium committed:
Add bluetooth bidi module

sadym-chromium committed:
Add documentation for extending BiDi

sadym-chromium committed:
Improve doc

OrKoN reviewed (commented):
Bluetooth tests need to be in the sub folder called external next to permissions tests. Also, I recommend separating the documentation updates from tests.

sadym-chromium committed:
Move bluetooth to external

sadym-chromium committed:
Rolls back doc

sadym-chromium commented:
> Bluetooth tests need to be in the sub folder called external next to permissions tests. 

Right, fixed.

> Also, I recommend separating the documentation updates from tests.

Moved doc to another PR

sadym-chromium committed:
fix flake8
